runtime/src/main.rs

```diff
@@ -79,6 +78,12 @@ pub extern "C" fn entry_point() -> ! {
         cprintln!("[state] CFI Disabled");
     }
 
+    drivers.run_reset_flow().unwrap_or_else(|e| {
+        cprintln!("[rt] Runtime failed reset flow");
+        handle_fatal_error(e.into());
+    });
+    caliptra_common::stop_wdt(&mut drivers.soc_ifc);
```

Comment:
I'm not sure why we had this so far up before. I would think we don't really want to stop the watchdog until we're about to enter the mailbox loop: https://github.com/chipsalliance/caliptra-sw/blob/9d8206e21edc2b89d435fc86dca38e7acb835819/runtime/src/lib.rs#L265

cc @korran in case I'm missing something.

Reply:
Done